data/bookkeeping/block.go

```diff
@@ -661,9 +675,18 @@ func (block Block) PaysetCommit() (TxnCommitments, error) {
 		}
 	}
 
+	var digestSHA512 crypto.Sha512Digest
+	if params.EnableSha512BlockHash {
+		digestSHA512, err = block.paysetCommitSHA512()
+		if err != nil {
+			return TxnCommitments{}, err
```

Comment:
This seems like could be a big problem if it's enabled - should we log an error somewhere?